.github/workflows/openapi-update-generate-new-sdk.yml

```diff
@@ -1,9 +1,9 @@
 name: Update bundled openapi definition & Generate SDK
 
 on:
-  push:
-    branches:
-      - main
+  #push:
+    #branches:
+      #- main
```

Comment:
this will be uncomented?

Reply:
yes, because it will run automatically and create a merge conflict for this PR.